frontend/src/components/organism/Projections.tsx

```diff
@@ -26,11 +26,14 @@ const Projections: React.FC<{ className?: string }> = ({ className }) => {
 
   const renderTransactions = () =>
     transactions
-      .filter(t =>
-        moment(t.date).isBetween(
-          moment().startOf('month'),
-          moment().add(5, 'years')
-        )
+      .filter(
+        t =>
+          moment(t.date).isSameOrAfter(moment().startOf('month')) &&
+          moment(t.date).isSameOrBefore(
+            moment()
+              .add(5, 'years')
+              .endOf('month')
+          )
```

Comment:
Moment.js actually has support for this built-in. I believe it might be better to use that, as it is already implemented.

```suggestion
moment(t.date).isBetween(
  moment(),
  moment().add(5, 'years'),
  'month',
  '[]'
)
```

The third param tells moment which _scope_ to use, so that it will ignore the days.

The fourth param tells it how it should handle inclusivity. The syntax is `(`/`)` for exclusivity, and `[`/`]` for inclusivity.

Reply:
[Source](https://momentjs.com/docs/#/query/is-between/)

Reply:
Thanks!